ee/hogai/graph/root/nodes.py

```diff
@@ -224,6 +224,7 @@ def _get_model(self, state: AssistantState, tools: list[RootTool]):
             max_tokens=8192,
             thinking=self.THINKING_CONFIG,
             conversation_start_dt=state.start_dt,
+            billable=True,
```

Comment:
As we said before, this shouldn't be billable if the tool calls are not all billable (e.g. search documents)